hack/lib/init.sh

```diff
@@ -50,8 +50,10 @@ os::log::stacktrace::install
 # them before every invocation.
 os::util::environment::update_path_var
 
-if [[ -z "${OS_TMP_ENV_SET-}" ]]; then
```

Comment:
We still want this condition as the top-level `if`

Reply:
fix it.